shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1129,7 +1129,8 @@ public void attachToFlutterEngine(@NonNull FlutterEngine flutterEngine) {
             this,
             this.flutterEngine.getTextInputChannel(),
             this.flutterEngine.getScribeChannel(),
-            this.flutterEngine.getPlatformViewsController());
+            this.flutterEngine.getPlatformViewsController(),
```

Comment:
Unrelated to your change- would probably be simpler to just pass flutterEngine into this call?